settings/src/components/totp.js

```diff
@@ -249,18 +211,27 @@ function SetupForm( { handleEnable, qrCodeUrl, secretKey, inputs, setInputs, err
 	const canSubmit = qrCodeUrl && secretKey && isInputComplete;
 
 	return (
-		<form className="wporg-2fa__setup-form" onSubmit={ handleEnable }>
-			<AutoTabbingInput inputs={inputs} setInputs={setInputs} error={error} onComplete={handleComplete}/>
-
-			<div className="wporg-2fa__submit-btn-wrapper">
-				<Button variant="secondary" onClick={ handleCancelClick }>
-					Cancel
-				</Button>
-				<Button type="submit" variant="primary" disabled={ ! canSubmit }>
-					Enable
-				</Button>
-			</div>
-		</form>
+		<Flex expanded={false} direction='column' align='center' gap="16px" className="wporg-2fa__setup-form-container">
+			<Notice status="error" isDismissible={ false } className={ error ? "show" : ""}>
```

Comment:
```suggestion
			<Notice status="error" isDismissible={ false } className={ error ? "is-shown" : ""}>
```

I think this might be more consistent with [Gutenberg naming standards](https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/#naming).

Reply:
Thanks 👍